config/clusters/nasa-cryo/common.values.yaml

```diff
@@ -95,53 +95,194 @@ basehub:
+              node.kubernetes.io/instance-type: r5.xlarge
+
+        - display_name: "Medium: up to 16 CPU / 128 GB RAM"
+          description: *profile_list_description
+          slug: medium
           allowed_teams:
             - 2i2c-org:hub-access-for-2i2c-staff
             - CryoInTheCloud:cryocloudadvanced
+          profile_options:
+            requests:
+              # NOTE: Node share choices are in active development, see comment
+              #       next to profileList: above.
+              display_name: Node share
+              choices:
+                mem_1:
+                  display_name: ~1 GB, ~0.125 CPU
+                  kubespawner_override:
+                    mem_guarantee: 0.942G
+                    cpu_guarantee: 0.013
+                mem_2:
+                  display_name: ~2 GB, ~0.25 CPU
+                  kubespawner_override:
+                    mem_guarantee: 1.883G
+                    cpu_guarantee: 0.025
+                mem_4:
+                  default: true
+                  display_name: ~4 GB, ~0.5 CPU
+                  kubespawner_override:
+                    mem_guarantee: 3.766G
+                    cpu_guarantee: 0.05
+                mem_8:
+                  display_name: ~8 GB, ~1.0 CPU
+                  kubespawner_override:
+                    mem_guarantee: 7.532G
+                    cpu_guarantee: 0.1
+                mem_16:
+                  display_name: ~16 GB, ~2.0 CPU
+                  kubespawner_override:
+                    mem_guarantee: 15.064G
+                    cpu_guarantee: 0.2
+                mem_32:
+                  display_name: ~32 GB, ~4.0 CPU
+                  kubespawner_override:
+                    mem_guarantee: 30.128G
+                    cpu_guarantee: 0.4
+                mem_64:
+                  display_name: ~64 GB, ~8.0 CPU
+                  kubespawner_override:
+                    mem_guarantee: 60.257G
+                    cpu_guarantee: 0.8
+                mem_128:
+                  display_name: ~128 GB, ~16.0 CPU
+                  kubespawner_override:
+                    mem_guarantee: 120.513G
+                    cpu_guarantee: 1.6
           kubespawner_override:
+            cpu_limit: null
             mem_limit: null
-            mem_guarantee: 115G
             node_selector:
-              node.kubernetes.io/instance-type: m5.8xlarge
+              node.kubernetes.io/instance-type: r5.4xlarge
+
+        # NOTE: The large option is added as a comment for now. It may be that
+        #       its relevant in the future for advanced users having a workshop,
+        #       and then its possible to enable more easily.
+        #
+        #       This setup was discussed with Tasha Snow in March 2023 at
+        #       https://2i2c.freshdesk.com/a/tickets/543.
+        #
+        # - display_name: "Large: up to 64 CPU / 512 GB RAM"
+        #   description: *profile_list_description
+        #   slug: large
```

Comment:
Ditto if this comment is for reverting then the commit should just be revertable not commented out like this.

Reply:
I planned for the section to be de-commented in the future if requested. I comment out and de-comment blocks of lines regularly in vscode using `ctrl+k+c` and `ctrl+k+u` respectively.

I can create commits to make this revertable, or do so in the future if you wish!